src/test/java/SaucedemoTesting/StepDefinitions/TestBase.java

```diff
@@ -0,0 +1,28 @@
+package SaucedemoTesting.StepDefinitions;
+
+import org.openqa.selenium.JavascriptExecutor;
+import org.openqa.selenium.WebDriver;
+import org.SaucedemoTesting.DriverFactory;
+
+public class TestBase {
+
+    public WebDriver driver = DriverFactory.getDriver();
+
+    // useful when running tests in Firefox
```

Comment:
who says that?

Reply:
M-am inspirat dintr-un curs si nu am rulat decat pe chrome, dar o sa sterg daca nu imi foloseste.